tests/test_router.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from django.db.utils import ProgrammingError
+
+from asset_dashboard.models import Section, NaturePreserves
+
+
+@pytest.mark.django_db(databases=['default', 'fp_postgis'])
+def test_gis_query(nature_preserves):
+    assert NaturePreserves.objects.db == 'fp_postgis'
```

Comment:
If `using` is not passed, this attribute is set by the `db_for_read()` method of the router. See: https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/db/models/manager.py#L135